eng/Versions.props

```diff
@@ -38,7 +38,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependency from https://github.com/dotnet/test-templates -->
-    <MicrosoftDotNetTestProjectTemplates30PackageVersion>1.0.2-beta4.22406.1</MicrosoftDotNetTestProjectTemplates30PackageVersion>
+    <!-- Version for netcoreapp2.1 is also shipped because it is inserted into VS.Test-templates repository no longer produces that template
```

Comment:
```suggestion
    <!-- Version for netcoreapp2.1 is also shipped because it is inserted into VS. Test-templates repository no longer produces that template
```